smartsim/exp/ray/raycluster.py

```diff
@@ -129,12 +146,23 @@ def __init__(
             interface=interface,
             alloc=alloc,
             num_nodes=num_nodes,
+            run_command=run_command if run_command else "auto",
+            host_list=host_list,
             **kwargs,
         )
         if batch:
-            self._build_batch_settings(num_nodes, time, batch_args, launcher)
+            self.batch_settings = create_batch_settings(
```

Comment:
Assuming that the previous ``SSUnsupportedError`` catches the cases where ``create_run_settings`` may return an incompatible run command?

Reply:
I cannot think of a case in which this happens and it is not caught. I might be proven wrong, but all test cases passed.